commands/download.go

```diff
@@ -0,0 +1,72 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	faasCmd.AddCommand(downloadCmd)
+}
+
+var downloadCmd = &cobra.Command{
+	Use:     `download`,
+	Short:   "Download binaries",
+	Long:    "Download binaries",
+	Example: "faas-cli download",
+	Run:     runFunc,
+}
+
+func runFunc(cmd *cobra.Command, args []string) {
```

Comment:
I don't think the `runFunc` as it is currently implemented the help text looks like 

```sh
Download binaries

Usage:
  faas-cli download [flags]
  faas-cli download [command]

Examples:
faas-cli download

Available Commands:
  kubeseal    Download kubeseal

Flags:
  -h, --help   help for download

Global Flags:
      --filter string   Wildcard to match with function names in YAML file
      --regex string    Regex to match with function names in YAML file
  -f, --yaml string     Path to YAML file describing function(s)

Use "faas-cli download [command] --help" for more information about a command.
You can download: kubeseal
```

The last line duplicates the content of the `Available Commands` section

But if we leverage the default sub-command help text behavior we essentially the same thing with a lower long term maintenance as we add more downloads

```sh
Download binaries

Usage:
  faas-cli download [command]

Examples:
faas-cli download

Available Commands:
  kubeseal    Download kubeseal

Flags:
  -h, --help   help for download

Global Flags:
      --filter string   Wildcard to match with function names in YAML file
      --regex string    Regex to match with function names in YAML file
  -f, --yaml string     Path to YAML file describing function(s)

Use "faas-cli download [command] --help" for more information about a command.
```